org.garudalinux.firedragon.yml

```diff
@@ -0,0 +1,74 @@
+app-id: org.garudalinux.firedragon
+runtime: org.freedesktop.Platform
+runtime-version: 23.08
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: 23.08
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: 23.08
+    add-ld-path: .
+command: firedragon
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --env=GTK_PATH=/app/lib/gtkmodules
```

Comment:
```suggestion
```

This is not required anymore, the path is empty. It used to be, a long time ago.

Reply:
Fixed in 986934eb8557f7b170b189a81f9d9f3c5d273090